Sources/Benchmark/Benchmark.swift

```diff
@@ -31,6 +34,12 @@ internal class ClosureBenchmark: AnyBenchmark {
     }
 }
 
-public func benchmark(_ name: String, f: @escaping () -> Void) {
-    defaultBenchmarkSuite.benchmark(name, f)
+public func benchmark(_ name: String, using f: @escaping () -> Void) {
+    defaultBenchmarkSuite.benchmark(name, function: f)
+}
+
+public func benchmark(
+    _ name: String, settings: BenchmarkSetting..., function f: @escaping () -> Void
+) {
+    defaultBenchmarkSuite.benchmark(name, settings: settings, function: f)
```

Comment:
```suggestion
    _ name: String, settings: BenchmarkSetting..., function: @escaping () -> Void
) {
    defaultBenchmarkSuite.benchmark(name, settings: settings, function: function)
```

Note: I think we could probably come up with a better name than `function` (as that's just the type), but I think we can defer this for future haggling.